packages/expo-insights/ios/InsightsModule.swift

```diff
@@ -7,6 +7,20 @@ import EASClient
 private var wasAppLaunchEventDispatched = false
 
 public final class InsightsModule: Module {
+  #if os(iOS)
```

Comment:
@reichhartd here is a (hopefully future proof) adjustment for the platform string value -- thanks for finding that!